.gitlab-ci.yml

```diff
@@ -2580,3 +2631,31 @@ stages:
     JANE_INVENTORY_HOSTVARS: 'yadm__dotfiles_enabled=true'
     JANE_DIFF_PATTERN: '.*/roles/yadm/.*'
     JANE_LOG_PATTERN: '\[yadm\]'
+
+
+# === Ansible no dependencies roles === [[[2
+
+'ferm role':
```

Comment:
I'm moving the alphabetical list to number of dependencies lists, with inner alphabetical organisation. For some reason I had errors that the `needs` dependencies were not predeclared.